jnosql-metamodel-processor-extension/README.adoc

```diff
@@ -0,0 +1,31 @@
+= Metamodel Processor
+
+This extension contains the Metamodel processor which, using the Metamodel layer, is able to automatically create description of a type annotated with JNoSql annotations such as Entity, MappedSupertype or Embeddable.
+Include this library as an optional dependency to automatically activate the metamodel generation at compile time.
+
+So, for example :
+
+[source,java]
+----
+@Entity
+public class Person {
+
+    @Id
+    private String name;
+
+    @Column
+    private Integer age;
+
+}
+----
+
+Will produce this Metamodel :
+
+[source,java]
+----
+public class Person_ {
+
+    public static volatile StringAttribute<org.eclipse.jnosql.mapping.metamodel.processor.Person> name = new DefaultStringAttribute(org.eclipse.jnosql.mapping.mongodb.Person.class, "name");
+    public static volatile NumberAttribute<org.eclipse.jnosql.mapping.metamodel.processor.Person, Integer> age = new DefaultNumberAttribute(org.eclipse.jnosql.mapping.mongodb.Person.class, Integer.class, "age");
```

Comment:
See prior comment about specification packages and ambiguity between `Person` classes, which also applies here.

Reply:
Thank you ! Missed the wrong package reference.
I fixed that, now only the correct one is used, the one that is actually provided in test packages.